homeassistant/components/nice_go/coordinator.py

```diff
@@ -178,8 +178,8 @@ async def _update_refresh_token(self) -> None:
 
     async def client_listen(self) -> None:
         """Listen to the websocket for updates."""
-        self.api.event(self.on_connected)
-        self.api.event(self.on_data)
+        self.api.listen("on_connected", self.on_connected)
+        self.api.listen("on_data", self.on_data)
```

Comment:
Should we also remove these listeners?

Reply:
I believe so, couldn't find a way to detect when to remove them

Reply:
I will mention this doesn't tell the API to listen for that event and it only tells the library what function to call when an event from a subscription happens